### ISSUE TITLE
Maven Artifcat

### ISSUE THREAD

**Issue by slekha** [janicki/TACACS#7]
Hi,
The jar file generated i.e tacacs.jar doe not seem to be available in Maven Repositories. I am evaluating this for use in a product and looks like a good fit for the requirements we have. Is there any reason why this is not uploaded to maven repositories?
Thank you!

**Comment by janicki** [comment]
Hi,  I’m glad you’re finding the TACACS project useful.  Uploading to a Maven repository sounds good… I’ll get back to you later this week.

**Comment by slekha** [comment]
Thank you!

**Comment by slekha** [comment]
Hi,
Following up on this , is this being considered for maven repository upload?

**Comment by janicki** [comment]
Yes sorry, just had a busy week.  I'll try this weekend or early next week.

**Comment by AugurSystems** [comment]
Here it is: 
https://raw.github.com/AugurSystems/TACACS/repository/

**Comment by markusju** [comment]
@janicki @AugurSystems I very much appreciate your work and would also like to use the library over maven. Unfortunately, I can't get the link above to work. 
I have forked this repo and slightly adjusted the build environment to support maven. Using jitpack.io it can be used in maven. I thought that others might find this useful:

```
	<repositories>
		<repository>
		    <id>jitpack.io</id>
		    <url>https://jitpack.io</url>
		</repository>
	</repositories>
...
	<dependency>
	    <groupId>com.github.markusju</groupId>
	    <artifactId>TACACS</artifactId>
	    <version>Tag</version>
	</dependency>
```

More Information on jitpack.io can be found here: https://jitpack.io/#markusju/TACACS/Tag

**Comment by janicki** [comment]
Hi Markus,  I added [this section](https://github.com/AugurSystems/TACACS/blob/master/README.md#gradle-maven-sbt-leiningen) to the project description.  Let me know if you would prefer anything changed.  Thanks for sharing your work!
-- Chris
